ObservatoryCore/UI/CoreForm.Settings.cs

```diff
@@ -91,7 +92,11 @@ private void PopulateDropdownOptions()
             var voices = new System.Speech.Synthesis.SpeechSynthesizer().GetInstalledVoices();
             foreach (var voice in voices.Select(v => v.VoiceInfo.Name))
                 VoiceDropdown.Items.Add(voice);
-#endif 
+
+            for (int n = -1; n < WaveOut.DeviceCount; n++)
+                AudioDeviceDropdown.Items.Add(WaveOut.GetCapabilities(n).ProductName);
+            AudioDeviceDropdown.SelectedIndex = Properties.Core.Default.AudioDevice + 1; // The +1 accounts for the device ID having -1 as a possible value
```

Comment:
What happens if a user's hardware config changes and has a different number audio devices connected than the last time they used Observatory? Or even if they haven't changed are we sure the order of devices is stable?